gradle.properties

```diff
@@ -6,4 +6,4 @@ deployUrl = https://oss.sonatype.org/service/local/staging/deploy/maven2/
 # Enable the daemon by adding org.gradle.daemon in USER_HOME/.gradle/gradle.properties
 org.gradle.parallel=true
 
-java6Home=/Library/Java/JavaVirtualMachines/1.6.0_65-b14-462.jdk/Contents/Home
+java6Home=/Library/Java/JavaVirtualMachines/1.6.0.jdk/Contents/Home
```

Comment:
I've commited this line by mistake in a previous commit, but perhaps we can leave it there and set it to `1.6.0.jdk` so that devs just needs to set a symlink from `1.6.0.jdk` to their jdk6 version.
On mac, if installed from brew cask, default folder is `1.6.0.jdk` anyway.